### PR TITLE
SAMZA-1287: Updated instructions for running integration and stability tests.

### PR DESCRIPTION
**Pull request by prateekm**

### CODE REVIEW

**Comment by prateekm** [comment]
@navina @vjagadish1989, please take a look.

File: samza-test/src/main/config/join/README [hunk 1]
```diff
@@ -1,77 +1,86 @@
...
+> export SAMZA_SRC=/path/to/samza
+> export HELLO_SAMZA_SRC=/path/to/samza-hello-samza
+> export DEPLOY_DIR=$HELLO_SAMZA_SRC/deploy
 
-The steps to setup the integration tests is similar to the samza-hello-samza setup, using the grid script.
-Once you deploy Zookeeper, YARN and Kafka, you have to generate the tar ball with the tests.
-Before generating the tar ball, update the "yarn.package.path" in $SAMZA_SRC/src/config/join/common.properties file to the path where the published tar will be made available.
+Deploy Zookeeper, YARN and Kafka:
+> cd $HELLO_SAMZA_SRC
+> bin/grid bootstrap
```
Comment (navina) [inline samza-test/src/main/config/join/README]: *bootstrap* also downloads and installs *samza*. Is that necessary? If not, please correct the command. If yes, please fix Line 43.

File: samza-test/src/main/config/join/README [hunk 2]
```diff
@@ -1,77 +1,86 @@
...
 
---> Release the tar
-cd $SAMZA_SRC
-./gradlew releaseTestJobs
-cp ~/samza-test/build/distributions/samza-test_*.tgz $DEPLOY_DIR
-tar -xvf $DEPLOY_DIR/samza-test_*.tgz -C $DEPLOY_DIR/samza
+Generate the tarball with the test jobs:
```
Comment (navina) [inline samza-test/src/main/config/join/README]: Does Line 47 serve as title for lines 48 - 51 or 48 - 58? You are not actually generating any tar until Line 55. Kind of confusing.

File: samza-test/src/main/config/join/README (hunk 1 above)
Comment (prateekm) [inline samza-test/src/main/config/join/README]: It's not necessary, but it doesn't hurt. IIUC the alternative is to chain 4 different commands:
bin/grid install kafka && bin/grid install yarn && bin/grid install zookeeper && bin/grid start all

Do you prefer one over the other?

File: samza-test/src/main/config/join/README (hunk 2 above)
Comment (prateekm) [inline samza-test/src/main/config/join/README]: Lines 48-58 with 2 sub-section. Can remove this line if that makes it clearer. WDYT?

File: samza-test/src/main/config/join/README (hunk 1 above)
Comment (navina) [inline samza-test/src/main/config/join/README]: I prefer instructions that are clear and don't have unwanted output/results :D 

I would just add this command (until we update the grid script):
`for i in zookeeper kafka yarn; do ./bin/grid install $i; ./bin/grid start $i; done`

File: samza-test/src/main/config/join/README (hunk 2 above)
Comment (navina) [inline samza-test/src/main/config/join/README]: Yeah. You can remove it or use markdown formatting to indicate heading level. I prefer the latter.

**Comment by prateekm** [comment]
@navina Updated with your feedback. Please merge if this looks OK. Let's keep converting to Markdown for later.

**Review by vjagadish1989 — APPROVED** [review]
looks good to me. approved

**Comment by vjagadish1989** [comment]
Thanks Prateek!

**Pull request closed without merging**